services/minespace-web/src/components/pages/Project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useState } from "react";
```

Comment:
For the new project description overview tab I think it is meant to be for both Core and Minespace? I am noticing the ticket doesn't specify, but I know the new submission page I added was for both Core and Minespace and is meant to redirect to the project description overview tab.

May have to check with Rebecca or Denise I think?